model_analyzer/triton/server/server_docker.py

```diff
@@ -142,7 +146,8 @@ def start(self, env=None):
                 tty=False,
                 stdin_open=False,
                 detach=True,
-                shm_size=self._shm_size)
+                shm_size=self._shm_size,
+                **self._args)
```

Comment:
Is there anything that makes sure the same arg isn't passed in twice? What if they pass in shm_size through the new option instead of the old one?

Reply:
Nothing explicit that I've done since this will cause an exception that says: `run() got multiple values for keyword argument 'shm_size'

Felt like that was good enough and a user should be able to figure out what they've done wrong.